src/site/content/en/lighthouse-performance/dom-size/index.md

```diff
@@ -8,22 +8,57 @@ web_lighthouse:
   - dom-size
 ---
 
```

Comment:
Sg.

Reply:
Gonna ping you with a question.

Reply:
Adding it here, as well. Lighthouse has two titles-- one for passing audit and one for failing audit.

I'm gonna use failure titles, as that seems the closest match to search situations.

So in above example, I'd use "Avoid an excessive DOM size" instead of "Avoids an excessive DOM size".

Make sense? Just want to confirm.